x-pack/plugins/threat_intelligence/public/modules/indicators/lib/display_value.ts

```diff
@@ -41,44 +190,32 @@ const rulesArray: MapperRule[] = [
   [isIpIndicator, extractIp],
   [isUrlIndicator, extractUrl],
   [isFileIndicator, extractFile],
+  [isUnknownIndicator, extractId],
+  [isEmailAddress, extractEmailAddress],
+  [isDomain, extractDomain],
+  [isX509Certificate, extractX509Serial],
+  [isWindowsRegistryKey, extractWindowsRegistryKey],
+  [isAutonomousSystem, extractAutonomousSystemNumber],
+  [isMacAddress, extractMacAddress],
 ];
 
 /**
  * Finds display value mapping function for given indicatorType
  */
-const findMappingRule = (indicatorType: string | null): IndicatorValueExtractor => {
-  const [_, extract = extractStub] = rulesArray.find(([check]) => check(indicatorType)) || [];
+const findMappingRule = (indicatorType: string | null): IndicatorDisplayNameExtractor => {
+  const [_, extract = extractId] = rulesArray.find(([check]) => check(indicatorType)) || [];
```

Comment:
defaulting to ID extractor, I think it makes sense to use _id for the display value for an unknown IoC type. wdyt?